README.md

```diff
@@ -46,6 +46,21 @@ Directly from unpkg.com
 </body>
 ```
 
+## Rendering multiple banners with one slot ID
+
+You can render multiple banners using the same slot ID and dimensions by setting up
+a banner context. This is useful when you want to run an auction with multiple results.
+To do that you have to use the `topsort-banner-context` and `topsort-banner-slot` elements.
+`topsort-banner-context` takes the same properties as the regular `topsort-banner` element.
+
+```html
+<topsort-banner-context width="600" height="400" id="<your slot id>">
```

Comment:
Can we keep the original `<topsort-banner>` element instead of making a new one, and then look for a sub-tag (similar to what you have; perhaps `<topsort-winner rank="1">`? The api feels easier to understand

Reply:
Ideally I would use the same element and then pass different attrs depending on context. But Idk how to do that

Reply:
Thinking that we'd iterate the sub elements and look for `<topsort-winner rank="1">` before deciding what logic to apply?

Reply:
Easier said than done, as there is no easy way for a component to check its children if they have any, as everything is built in an asynchronous manner. So at some point in time the banner could have no children, and then it does. Same goes with attributes: Components attributes are `undefined` on construction and then defined after it is mounted in the Shadow DOM.

Reply:
After some more digging and  experimentation I manage to do it with one component. Tho is more complicated this way, at least we don't have to maintain 3 components.